config.go

```diff
@@ -136,6 +136,13 @@ func (b *Builder) initConfig() {
 		b.OCIv1 = image
 		b.Docker = dimage
 	}
+	if len(b.Manifest) > 0 {
```

Comment:
If we've a docker image, will b.Manifest still be > 0 ?  Are the manifests the same regardless of having a docker or  ociv1 image so no special handling is necessary in the below unmarshalling?